src/renderer/rendererutils.ts

```diff
@@ -36,22 +19,26 @@ const getInstanceDifficulty = (difficultyID: number): InstanceDifficultyType | n
         return "";
     }
 
-    const isPvp = isCategoryPVP(category);
-    let resultText: string;
+    switch (category) {
+        case VideoCategory.MythicPlus:
+            return isGoodResult ? "Timed" : "Depleted";
 
-    if (isPvp && isGoodResult) {
-        resultText = "Win";     
-    } else if (isPvp && !isGoodResult) {
-        resultText = "Loss";
-    } else if (!isPvp && isGoodResult) {
-        resultText = "Kill";
-    } else {
-        resultText = "Wipe";
-    }
+        case VideoCategory.Raids:
+            return isGoodResult ? "Kill" : "Wipe";
 
-    return resultText;
+        default:
+            return isGoodResult ? "Win" : "Loss";
+    }
 } 
 
+const getVideoResult = (video: any): boolean => {
+    if (video.challengeMode !== undefined) {
```

Comment:
Why does this need special handling? It is still a boolean at the end of the day?

Reply:
You mean why ChallengeMode need its own handling or why I'm testing for `undefined`?

Reply:
Both I suppose. Why isn't video.result === video.challengeMode.timed in this context?

Reply:
Because a dungeon can be completed, but not timed. If a dungeon isn't completed, `video.result` will be false, if it's completed by not timed `video.result = true` and `video.challengeMode.timed = false`.

Reply:
Obvious really... thanks!